scripts/g.extension/g.extension.py

```diff
@@ -1296,8 +1296,17 @@ def get_multi_addon_addons_which_install_only_html_man_page():
     )
     if not os.path.exists(addons_paths_file):
         get_addons_paths(gg_addons_base_dir=options["prefix"])
-    with open(addons_paths_file) as f:
-        addons_paths = json.loads(f.read())
+    try:
+        with open(addons_paths_file) as f:
+            addons_paths = json.loads(f.read())
+    except EnvironmentError:
+        gscript.warning(
+            _(
+                "It is not possible to determine which multi-addons install"
+                " only the manual HTML page, because the <{}> does not exist."
```

Comment:
The relevance of the message to the user is not clear. Will re-running help? Should I get the file somewhere? Is that a problem? Perhaps add "Try running the installation again."

Reply:
I think we don't need too complicated printing of warning messages, we won't print any message here. However, we can edit the previous warning message and add information about the time when the GitHub API will be available again (get time information from the response)


Something like that:

From 

```
WARNING: The download of the json file with add-ons paths from the github
         server wasn't successful, rate limit exceeded. The previous
         downloaded json file will be used if exists.
```

to 

```
WARNING: The download of the JSON file with add-ons paths from the GitHub API
         server wasn't successful, rate limit exceeded. Try addon installation again
         at 08:45.
```

Reply:
This should be resolved in the new PR.